Client/Frontend/Browser/New Tab Page/Sections/BraveNewsSectionProvider.swift

```diff
@@ -117,23 +120,84 @@ class BraveNewsSectionProvider: NSObject, NTPObservableSectionProvider {
         return 20
     }
     
+    private var iabTrackedCellContexts: [IndexPath: ViewportTrackedCardContext] = [:]
+    
+    /// Information about a IAB tracked card to determine if a user viewed the ad by scrolling
+    /// at least 50% of the cell into the viewport for at least 1 second
```

Comment:
crazy, can QA verify it or a new debug entry needs to be added?

Reply:
They can verify in maybe two ways:
- Logs, there are unique logs that appear when a successful view happens
- Confirmations, I believe viewing an ad with Rewards enabled counts as "viewing" an ad, and thus you would get rewarded and the ad count/estimated payout would increase. Would have to check with Terry on this